src/styles/aui/_aui.navigation-menu.scss

```
@@ -0,0 +1,427 @@
+        text-align: center;
+      }
+
+      .a-icon {
+        font-size: $icon-size-base;
+        margin-bottom: $spacer-xs / 2;
+
+        &.a-tab-icon {
+          width: $icon-size-lg;
+        }
+
+        &.a-tab-submenu-icon {
+          display: none;
+          transition: transform $animation-speed ease;
+
+          @include at($screen-sm) {
+            display: inline;
+          }
+        }
+
+        @include at($screen-sm) {
+          margin-bottom: 0;
+        }
+      }
+
+      @include at($screen-sm) {
+        align-items: center;
+        flex-direction: row;
+        height: $header-height;
+        justify-content: space-between;
+        padding: $spacer-xs;
+        position: relative;
+
+        .o-menu__link {
+          margin-left: $spacer-sm;
+        }
+
+        .o-menu-tab__title {
+          align-items: center;
+          display: flex;
+          flex-direction: row;
+        }
+      }
+    }
+  }
+}
+
+.o-menu .o-menu-submenu {
+  display: none;
+
+  .u-cursor-pointer {
+    cursor: pointer;
+  }
+
+  .o-sub-menu__content-wrapper {
+    background: $white;
+    display: none;
+    max-height: 0;
+    overflow: hidden;
+    transition: max-height ease-in-out $animation-speed;
+
+    .m-nav-list {
+      border: none;
+
+      li {
+        position: relative;
+      }
+
+      li > a {
+        border: none $border-color;
+        color: $text-color-light;
+        display: flex;
+        height: $header-height;
+        justify-content: space-between;
+        overflow: hidden;
+        white-space: nowrap;
+
+        @include at($screen-sm) {
+          // Padding is parent padding + icon size + header padding to the left to align the link to the header text
+          padding-left: calc(#{$menu-docked-width} + #{$submenu-inset});
+          padding-right: $spacer-xs / 2;
+
+          a {
+            border: none;
+          }
+        }
+      }
+
+      li > a > .a-icon {
+        transform: rotate(0deg);
+      }
+    }
+
+    @include at($screen-sm) {
+      li > a > .a-icon {
+        margin-right: $spacer-xs /2;
+        transform: rotate(0deg);
+        transition: transform $animation-speed ease-in-out;
+      }
+
+      li.active {
+        & > .o-menu-submenu > .o-sub-menu__content-wrapper {
+          display: block;
+          max-height: $submenu-height;
+          overflow: visible;
+        }
+
+        & > a > .a-icon {
+          transform: rotate(180deg);
+        }
+      }
+    }
+  }
+
+  // First sub-layer of menus will get larger padding
+  & .m-nav-list li .o-menu-submenu li > a {
+    padding-left: calc(#{$menu-docked-width} + #{$submenu-inset * 2});
+  }
+
+  @include at($screen-sm) {
+    display: block;
+  }
+}
+
+.o-menu > .o-menu__hide-cta {
+  display: none;
+
+  @include at($screen-sm) {
+    display: block;
+    margin: $spacer-sm;
+    text-align: right;
+
+    > a {
+      text-decoration: none;
+    }
+  }
+}
+
+.o-menu > .o-menu-navigation-pane {
```

```suggestion
.o-menu > .o-menu__navigation-pane {
```
Is die direct child arrow trouwens nodig?